README.md

```diff
@@ -0,0 +1,3 @@
+# FOOD EXPIRATION MANAGER 
```

Comment:
Please change this title with the title of the project: _Food Expiration Dates_

Reply:
ok no problem